js/main.js

```diff
@@ -272,19 +272,26 @@ $(document).ready(function() {
         };
 
         var set_contents = function(e, to_set){
-          if(Boolean(e))
+          if(Boolean(e)) {
             e.innerHTML = to_set;
-        }
-
-        for (var key in dataArray) {
-          var value = dataArray[key];
-          var ref = document.getElementsByClassName(key);
-          // Some pages have more than one member counter
-          for (var i = 0, len = ref.length; i < len; i++) {
-            var element = ref[i];
-            set_contents(element, value);
+            console.log("Setting", e, "to", to_set);
           }
-        }
+        };
+
+        _.map(classValueMap, function(val, className){
+          _.map(document.getElementsByClassName(className), function(elem){
+            set_contents(elem, val);
+          });
+        });
```

Comment:
Haha thanks, this is much more succinct than the code I converted from CoffeeScript :P I should really try to work with underscore a bit more. 👍